pkg/cli/config_cmd.go

```diff
@@ -204,6 +204,10 @@ func addConfig(configPath, configName, url string) error {
 		return zotErrors.ErrInvalidURL
 	}
 
+	if checkConfigName(configs, configName) {
```

Comment:
try a name like `configNameExists` so it is clear in which case the function returns true/false from the name itself.